src/status_im/contexts/wallet/events.cljs

```diff
@@ -240,10 +242,32 @@
                                                  accounts))
                                              stored-accounts
                                              tokens-per-account))]
-     {:db (-> db
+     {:fx [[:dispatch [:wallet/get-last-wallet-token-update-if-needed]]]
+      :db (-> db
               (update-in [:wallet :accounts] add-tokens tokens)
               (assoc-in [:wallet :ui :tokens-loading address] false))})))
 
+(rf/reg-event-fx
+ :wallet/get-last-wallet-token-update-if-needed
+ (fn [{:keys [db]}]
+   (when (->> (get-in db [:wallet :ui :tokens-loading])
```

Comment:
I'd give a name to this condition to make understanding easier. Something like `all -tokens-loaded?`